### PR TITLE
Date element added to options page

### PR DESCRIPTION
**Pull request by RohitKaushal7**
- **What does this PR do?**
- Date Element added to the options page saved scrrolls.
 - files changed (for date) : save.js , options.js, options.css.
- +some design and animation changes. 

- **Screenshots and/or Live Demo**
image

### CODE REVIEW

**Comment by prateek3255** [comment]
@RohitKaushal7 Let's not add the gradient to the background because currently it matches the overall extension theme.

File: save.js [hunk 1]
```diff
@@ -7,10 +7,13 @@
     const offset = window.pageYOffset;
     const total = document.body.scrollHeight;
     const title = document.title;
+    let date = new Date;
+    let adate = String(date).split(' ');
+    let sdate = `${adate[0]} ${adate[2]} ${adate[1]} ${adate[3]}`;
```
Comment (prateek3255) [inline save.js]: Please use some better variable names here.

Comment (RohitKaushal7) [reply save.js]: ok.. :)

**Comment by RohitKaushal7** [comment]
@prateek3255 I have made the variable changes and restored the style changes (/* some commented out */), although there are still some hover animation (style changes). please remove the code lines what you feel doesn't suit.

**Comment by RohitKaushal7** [comment]
Merged The Delete options and some style changes...

demo..

deepin-screen-recorder_Select area_20191030230032

**Comment by prateek3255** [comment]
LGTM! Thanks for working on this @RohitKaushal7

**Pull request merged**